frontend/src/_components/LogoNavDropdown.jsx

```diff
@@ -20,7 +20,12 @@ export default function LogoNavDropdown({ darkMode }) {
     const { admin } = authenticationService?.currentSessionValue ?? {};
     return (
       <div className={`logo-nav-card settings-card card ${darkMode && 'dark-theme'}`}>
-        <Link className="dropdown-item tj-text tj-text-xsm" data-cy="back-to-app-option" onClick={handleBackClick}>
+        <Link
+          className="dropdown-item tj-text tj-text-xsm"
+          data-cy="back-to-app-option"
+          onClick={handleBackClick}
+          to={DashboardUrl()}
```

Comment:
you can use getPrivateRoute('dashboard') instead of changing the existing fn.
```suggestion
          to={getPrivateRoute('dashboard')}
```

Reply:
Should I undo the changes in the existing fn ?

Reply:
You can try this function to get the route. But you'll need to test this out with subpath deployment.